clash-lib/src/Clash/Netlist/BlackBox/Util.hs

```diff
@@ -145,53 +145,55 @@ extractLiterals = map (\case (e,_,_) -> e)
 setSym
   :: forall m
    . Monad m
-  => (IdType -> Identifier -> m Identifier)
+  => (IdentifierType -> Data.Text.Text -> m Identifier)
```

Comment:
Perhaps make a type alias:
```
type IdentifierText = Text
```
and use that where the old code used to use `Identifier`